src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List, Optional
+
+import pytest
+
+from pants.backend.python.lint.bandit.rules import BanditTarget
+from pants.backend.python.lint.bandit.rules import rules as bandit_rules
+from pants.backend.python.rules import download_pex_bin, pex
+from pants.backend.python.subsystems import python_native_code, subprocess_environment
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.build_graph.address import Address
+from pants.build_graph.build_file_aliases import BuildFileAliases
+from pants.engine.fs import FileContent, InputFilesContent, Snapshot
+from pants.engine.legacy.structs import PythonTargetAdaptor
+from pants.engine.rules import RootRule
+from pants.engine.selectors import Params
+from pants.rules.core.lint import LintResult
+from pants.source.wrapped_globs import EagerFilesetWithSpec
+from pants.testutil.option.util import create_options_bootstrapper
+from pants.testutil.test_base import TestBase
+
+
+class BanditIntegrationTest(TestBase):
+  good_source = FileContent(path="test/good.py", content=b"hashlib.sha256()\n")
+  bad_source = FileContent(path="test/bad.py", content=b"hashlib.md5()\n")
+  # MD5 is a insecure hashing function
+
```

Comment:
tiny nit: I'd move this explanatory comment (which is good to have) above the line it comments on